_sass/features.scss

```diff
@@ -14,4 +14,8 @@
     i {
         color: $dtiles_blue;
     }
+}
+
+.feature-jobs { 
+    max-width: 250px;
```

Comment:
is the max-width needed? without it the line would just break and one of the openings would be in a second line, right?